packages/hardhat/contracts/QuadraticDiplomacyContract.sol

```diff
@@ -31,6 +35,31 @@ contract QuadraticDiplomacyContract is Distributor, AccessControl {
         _;
     }
 
+    // Warning: Only use in combination with onlyAdmin. The caller will get any leftovers.
```

Comment:
Question: Is it possible to use a modifier on a modifier?

Reply:
No, I dont think so 😞

Reply:
True!